.travis.yml

```diff
@@ -43,7 +43,6 @@ after_success:
   - |
     if [[ ${TRAVIS_PULL_REQUEST} == "false" ]]; then
       python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
-      export URL="https://userguide.mdanalysis.org/"
```

Comment:
Do you prefer to set the URL in the script instead of in the travis settings? You could add it to the global variables at the top of travis.yml as one of the major configuration settings. 

There are also still all the other variables that remain in the script. Why is URL moved?

Reply:
Probably just absentmindedness. This now needs to be replaced with something similar to the gh-actions pipeline in mdanalysis proper (https://github.com/MDAnalysis/mdanalysis/pull/3051)